releasenotes/jmri4.99.4.shtml

```diff
@@ -260,6 +260,11 @@ The <a href="https://github.com/JMRI/JMRI/issues?utf8=✓&q=milestone%3A4.99.4+i
         <a id="DecoderPro" name="DecoderPro"></a>
         <ul>
             <li>You can now write out just edited CV values to a CSV file.</li>
+            <li>The programmer pane "display" element now supports a "viewOnly" attribute.
+                For more information see the
+                <a href="/help/en/html/apps/DecoderPro/programmer.shtml" target="_blank">Create a Custom Programmer</a>
```

Comment:
This URL https://www.jmri.org/help/en/html/apps/DecoderPro/programmer.shtml gives me a 404.

But this URL works:
https://www.jmri.org/help/en/html/apps/DecoderPro/Programmer.shtml

The error seems to be that `programmer.shtml` should be `Programmer.shtml`.